tests/test_reinforce_algorithm.py

```diff
@@ -55,21 +54,28 @@ def __init__(self, size):
         self._estimates = np.random.uniform(size)
 
     def set_estimates(self, estimates):
-        self._estimates = estimates
+        self._estimates = np.array(estimates, dtype=np.float32)
 
     def estimate(self, _):
         return self._estimates
 
+    def fit(self, _):
+        pass
+
 
 class BaselineSpy(BaselineStub):
-    def estimate(self, _):
-        CALL_ORDER.append(self.estimate)
-        return super().estimate(_)
+    def __init__(self, size):
+        super().__init__(size)
+        self.received_returns = None
+        self.fit_onto = None
 
+    def estimate(self, trj):
+        self.received_returns = trj.returns
+        return super().estimate(trj)
 
-@pytest.fixture(autouse=True)
-def cleanup_call_order():
-    CALL_ORDER.clear()
+    def fit(self, trj):
+        super().fit(trj)
+        self.fit_onto = trj.returns
```

Comment:
This for sure will lead to confusion - in the Spy class, this is set to advantages, while here it is overridden with returns.

Reply:
Nope, this is the baseline spy so it works as expected